sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/command/DropTableSuite.scala

```diff
@@ -26,7 +26,7 @@ class DropTableSuite extends v1.DropTableSuiteBase with CommandSuiteBase {
   test("hive client calls") {
     withNamespaceAndTable("ns", "tbl") { t =>
       sql(s"CREATE TABLE $t (id int) $defaultUsing")
-      checkHiveClientCalls(expected = 11) {
+      checkHiveClientCalls(expected = 10) {
```

Comment:
Why this is changed?

Reply:
the v2 DROP TABLE command can save one PRC call.